### ISSUE TITLE
Builder v1 - cannot build salt for bookworm 

### ISSUE THREAD

**Issue by unman** [QubesOS/qubes-issues#9223]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
4.2

### Brief summary
As title

### Steps to reproduce
Set up build for 4.2 bookworm packages
make qubes-vm

### Expected behavior
Packages will be built

### Actual behavior
Build fails on salt-vm 

salt build-depends includes python3-looseversion, which is not available for bookworm, but is available for trixie.

**Comment by ben-grande** [comment]
> salt build-depends includes python3-looseversion, which is not available for bookworm, but is available for trixie.

Available on Trixie and Builder V1 doesn't seem to be used anymore and no builds of Debian 12 are being made since R4.3 release. Closing this as not planned, cause I don't think it was fixed or is planned to be fixed.

**Comment by unman** [comment]
On Wed, Feb 18, 2026 at 05:05:11AM -0800, ben-grande wrote:
> ben-grande left a comment (QubesOS/qubes-issues#9223)
> 
> > salt build-depends includes python3-looseversion, which is not available for bookworm, but is available for trixie.
> 
> Available on Trixie and Builder V1 doesn't seem to be used anymore and no builds of Debian 12 are being made since R4.3 release. Closing this as not planned, cause I don't think it was fixed or is planned to be fixed.
Absolutely agree about builder v1 - it's dead in the water without any
announcement.
BUT Debian-12 is still supported under 4.3 and I think the issue
remains. It's a build-requires but the package isnt available for
bookworm.

**Comment by ben-grande** [comment]
> BUT Debian-12 is still supported under 4.3 and I think the issue remains. It's a build-requires but the package isnt available for bookworm.

Still supported, but I don't think any effort will be done to port python3-looseversion to  Debian 12 on Qubes to support Salt. debian-12-minimal reports `os-eol` at 2026-06-10. Would you like to keep this bug open until then?

**Comment by unman** [comment]
I think we have not yet decided whether that will be eol for Qubes or we
will support LTS date.

**Comment by marmarek** [comment]
But python3-looseversion (for Debian 12) is provided via qubes repositories: https://github.com/QubesOS/qubes-python-looseversion, https://deb.qubes-os.org/r4.3/vm/pool/main/p/python-looseversion/
Salt is successfully built for bookworm: https://github.com/QubesOS/updates-status/issues/6187#issuecomment-3535504692
Am I missing something?

**Comment by ben-grande** [comment]
Nope... I didn't consult the Qubes deb repo... Can be closed as complated.

**Comment by andrewdavidwong** [comment]
Closing as completed. If anyone believes this issue is not yet completed, or if anyone is still affected by this issue, please leave a comment saying so, and we'll be happy to reopen it. Thank you.
